xapian-core/net/remoteserver.cc

```diff
@@ -255,6 +255,21 @@ RemoteServer::run()
 		case MSG_RECONSTRUCTTEXT:
 		    msg_reconstructtext(message);
 		    continue;
+		case MSG_SYNONYMTERMLIST:
+		    msg_getsynonymtermlist(message);
+		    continue;
+		case MSG_SYNONYMKEYLIST:
+		    msg_getsynonymkeylist(message);
```

Comment:
The other method names match the message names, but these two add `get`.  I'll adjust that as I merge.